### PR TITLE
Remaining files

### PR DESCRIPTION
**Pull request by billwambua**
dashboard_iccm files and dependencies.

After running docker-compose:

Selection_072

### CODE REVIEW

File: models/brac/views/formview_postnatal_care.sqlx [hunk 1]
```diff
@@ -1,43 +0,0 @@
-SELECT COALESCE((regexp_split_to_array((form.doc #>> '{fields,geolocation}'::text[]), ' '::text))[1], (form.doc #>> '{fields,location,lat}'::text[])) AS "inputs/meta/location/lat",
```
Comment (derickl) [inline models/brac/views/formview_postnatal_care.sqlx:1]: Are these files no longer needed?

Comment (billwambua) [reply models/brac/views/formview_postnatal_care.sqlx:1]: These files are not dependencies of any files. There are `useviews_` for many of these forms which select directly from couchdb so I removed these unneeded and redudant files.

File: models/brac/views/impactview_month_facility.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		SELECT
+			uuid AS facility_join_field,
+			name AS facility_name
+		FROM
+			{{ ref("contactview_metadata") }}
+		WHERE
+			type = 'district_hospital'
+			AND name <> 'HQ'
+			AND name <> 'HQ OVC'
+
```
Comment (derickl) [inline models/brac/views/impactview_month_facility.sql]: delete extra newlines

File: models/brac/views/impactview_month_facility.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	SELECT 
+		month,
+		epoch,
+		facility_join_field,
+		facility_name
+		
+	FROM
+		{{ ref("impactview_month") }}
+		CROSS JOIN facilities_CTE
+
```
Comment (derickl) [inline models/brac/views/impactview_month_facility.sql]: delete extra newlines

File: models/brac/views/impactview_month_facility.sql [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+			AND name <> 'HQ OVC'
+			
+	)
+	
+	SELECT 
+		month,
+		epoch,
+		facility_join_field,
+		facility_name
+
```
Comment (derickl) [inline models/brac/views/impactview_month_facility.sql]: delete extra newlines

File: models/brac/views/impactview_month.sql [hunk 5]
```diff
@@ -0,0 +1,5 @@
+SELECT 
+	 	generate_series(date_trunc('month', now() - '1 year'::interval), now(), '1 mon'::interval)::date AS month,
```
Comment (derickl) [inline models/brac/views/impactview_month.sql]: fix indentation

File: models/brac/views/impactview_month_facility.sql [hunk 6]
```diff
@@ -0,0 +1,27 @@
+WITH facilities_CTE AS
+	(
+		SELECT
+			uuid AS facility_join_field,
+			name AS facility_name
```
Comment (derickl) [inline models/brac/views/impactview_month_facility.sql]: fix indentation for this CTE

**Review by derickl — APPROVED** [review]
Nice!

Cosmetic feedback inline.

**Pull request merged**